src/applications/gqrx/remote_control.cpp

```diff
@@ -429,19 +429,24 @@ void RemoteControl::setMode(int mode)
 void RemoteControl::setNewRemoteFreq(qint64 freq)
 {
     qint64 delta = freq - rc_freq;
+    // it's an ugly but I hope this will be fixed with the issue #218
+    int FilterBW_half = std::min<int>(bw_half, 100000 /* WFM Wide */);
```

Comment:
Sorry, but with this patch the gqrx behaves better. If the total bandwidth is small we sets zero always as in original code.
Try it https://gist.github.com/kukabu/2bad6584291b9da24a36c4b091d0d46d (or something similar) with and without patch.